frontend/src/functions/filterAliases.test.ts

```diff
@@ -4,12 +4,15 @@ import {
   CustomAliasData,
   RandomAliasData,
 } from "../hooks/api/aliases";
-import { ProfileData } from "../hooks/api/profile";
 import { filterAliases, Filters } from "./filterAliases";
 
 const getMockedAliasMatching = (filters: Partial<Filters>): AliasData => {
   const alias: CustomAliasData | RandomAliasData = {
     address: filters.string ?? "arbitrary_string",
+    full_address:
+      filters.domainType === "custom"
+        ? `${filters.string ?? "arbitrary_string"}@some.mozmail.com`
```

Comment:
This is more of a personal preference, so feel free to disregard, but I usually try to keep the things that I'm asserting on in a unit test, inside that unit test. Whereas in this case, the caller of `getMockedAliasMatching` will have to _know_ that this function creates an alias with the subdomain `some.mozmail.com`, making the test harder to follow.

So I'd probably do something like adding an additional, optional `customSubdomain` parameter to this function, so that the calling test can explicitly set the domain to `some`:

```suggestion
        ? `${filters.string ?? "arbitrary_string"}@${customSubdomain ?? "arbitrary_subdomain"}.mozmail.com`
```